e2e/integration/BUILD.bazel

```diff
@@ -51,6 +51,38 @@ ts_library(
     ]
 )
 
+ts_library(
+    name = "resource_creation_spec",
```

Comment:
@subotic I forgot to do this for the tests I added. Is there a way to add all files contained in this folder to Bazel, so that not every single file has to be configured individually?

Reply:
I'll try to do this in another PR.